### PR TITLE
Orange shop

### PR DESCRIPTION
**Pull request by sarafuentes3025**
Program number 7

### CODE REVIEW

File: src/test/java/com/sfuentes/OrangeShopTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  void calculateFifteenPercentage_decimalNumber() {
+    OrangeShop client = new OrangeShop();
+    double number = 500.50;
+    double expectedResult = 75.075;
+
+    assertEquals(expectedResult, client.calculateFifteenPercentage(number));
+  }
+
+  @Test
+  void calculateCustomerPayment_tenKilos() {
```
Comment (aandraca) [inline src/test/java/com/sfuentes/OrangeShopTest.java:28]: I think there is one scenario left for the method calculateCustomerPayment: when is less than 10 kilos

**Review by aandraca — CHANGES_REQUESTED** [review]
Also, find a way to add the out folder to the .gitignore
I suggest to use the https://gitignore.io/ and search for maven to ignore all maven related things

File: src/test/java/com/sfuentes/OrangeShopTest.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    assertEquals(expectedResult, client.calculateFifteenPercentage(number));
+  }
+
+  @Test
+  void calculateFifteenPercentage_decimalNumber() {
+    OrangeShop client = new OrangeShop();
+    double number = 500.50;
+    double expectedResult = 75.075;
+
+    assertEquals(expectedResult, client.calculateFifteenPercentage(number));
```
[the remark this answers is not in the document]
Comment (sarafuentes3025) [reply src/test/java/com/sfuentes/OrangeShopTest.java]: Resolved

File: src/test/java/com/sfuentes/OrangeShopTest.java (hunk 1 above)
Comment (sarafuentes3025) [reply src/test/java/com/sfuentes/OrangeShopTest.java]: Resolved

**Review by aandraca — APPROVED** [review]

**Pull request merged**